docs/index.md

```diff
@@ -99,3 +99,13 @@ limit 1;
 | 186.187.161.169 | 502    | /path/to/web/page | POST           | 12345         |
 +-----------------+--------+-------------------+----------------+---------------+
 ```
+
+## Detections as Code with Powerpipe
```

Comment:
Even though we don't include the OWASP screenshot here compared to the README.md, can you please add it to the top of this doc?